website/addons/dropbox/utils.py

```diff
@@ -79,7 +79,12 @@ def log(self, action, extra=None, save=False):
 
 
 def is_subdir(path, directory):
-    if not (path and directory):
+    if not path:
+        return False
+    # directory is root directory
+    if directory == '' or directory == '/':
```

Comment:
Then we might want to modify `clean_path` to handle `/` as a special case. 

Rationale is that we want to be very conservative with allowing access to subdirectories. If a logical error is made that resulted in the empty string being passed, e.g. `abort_if_not_subdir('sensitive-folder', '')`, we would want to abort with a 401 rather than allow access.